src/herbivoresAndCarnivores.js

```diff
@@ -2,14 +2,36 @@
 
 class Animal {
   // write your code here
+  constructor(name, health) {
```

Comment:
```suggestion
  constructor(name) {
```
You need only name in the constructor